Modules/Core/Common/include/itkBackwardDifferenceOperator.h

```diff
@@ -55,7 +55,7 @@ class ITK_TEMPLATE_EXPORT BackwardDifferenceOperator : public NeighborhoodOperat
   /** Run-time type information (and related methods). */
   itkTypeMacro(BackwardDifferenceOperator, NeighborhoodOperator);
 
-  using PixelType = typename Superclass::PixelType;
+  using PixelType = TPixel;
```

Comment:
It seems to me that this alias isn't really necessary, it is? Use code may use `BackwardDifferenceOperatorType::PixelType`, even without this alias, because it is inherited from the base class.

    class Base
    {
    public:
      using SomeType = int;
    };

    class Derived: public Base {};

    int main()
    {
      Derived::SomeType var{42};
    }

You see, in this example, it is unnecessary to add an `Derived::SomeType` alias.

Reply:
See #2567.